docs/languages/cpp.md

```diff
@@ -21,95 +21,112 @@ C/C++ support for Visual Studio Code is provided by a [Microsoft C/C++ extension
+### Check if you have a compiler installed
 
-C++ is a compiled language meaning your program's source code must be translated (compiled) before it can be run on your computer. VS Code is first and foremost an editor, and relies on command-line tools to do much of the development workflow. The C/C++ extension does not include a C++ compiler or debugger. You will need to install these tools or use those already installed on your computer.
+> **Note**: There may already be a C++ compiler and debugger provided by your academic or work development environment. Check with your instructors or colleagues for guidance on installing the recommended C++ toolset (compiler, debugger, project system, linter).
 
-There may already be a C++ compiler and debugger provided by your academic or work development environment. Check with your instructors or colleagues for guidance on installing the recommended C++ toolset (compiler, debugger, project system, linter).
+Common compilers that already come preinstalled on some platforms are the [GNU Compiler Collection](https://wikipedia.org/wiki/GNU_Compiler_Collection) (GCC) on Linux and the [Clang](https://wikipedia.org/wiki/Clang) tools with [Xcode](https://developer.apple.com/xcode/) on macOS.
 
-Some platforms, such as Linux or macOS, have a C++ compiler already installed. Most Linux distributions have the [GNU Compiler Collection](https://wikipedia.org/wiki/GNU_Compiler_Collection) (GCC) installed and macOS users can get the [Clang](https://wikipedia.org/wiki/Clang) tools with [Xcode](https://developer.apple.com/xcode/).
+To check if you already have them installed:
 
-### Check if you have a compiler installed
+1. Open a new VS Code terminal window using (`kb(workbench.action.terminal.new)`)
+2. Use the following command to check for the GCC compiler `g++`:
+    ```bash
+    g++ --version
+    ```
 
-Make sure your compiler executable is in your platform path (`%PATH` on Windows, `$PATH` on Linux and macOS) so that the C/C++ extension can find it. You can check availability of your C++ tools by opening the Integrated Terminal (`kb(workbench.action.terminal.toggleTerminal)`) in VS Code and trying to directly run the compiler.
+    Or this command for the Clang compiler `clang`:
 
-Checking for the GCC compiler `g++`:
+    ```bash
+    clang --version
+    ```
 
-```bash
-g++ --version
-```
+The output should show you the compiler version and details. If neither are found, make sure your compiler executable is in your platform path (`%PATH` on Windows, `$PATH` on Linux and macOS) so that the C/C++ extension can find it. Otherwise, use the instructions in the section below to install a compiler.
 
-Checking for the Clang compiler `clang`:
+### Install a compiler
 
-```bash
-clang --version
-```
+If you don't have a compiler installed, you can follow one of our installation tutorials:
+
+- Windows:
+<a class="next-topic-btn" href="/docs/cpp/config-msvc.md#_prerequisites">Installing MSVC Tutorial</a> or
```

Comment:
Not sure how this will render on the live site so I recommend just using the path to the topic without `.md` or the header link.

Reply:
fixed